techscreenr-angular/client/app/questions/questions.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
```

Comment:
@amartinelli12  Try to follow the [Angular.io](https://angular.io/guide/styleguide#import-line-spacing) style guide on how to organize imports. The first row of the imports should be directly from the Angular framework, the second row should be third-party imports (ex NGRX), and the third row of imports should be our own code.